packages/dnd-timeline/src/hooks/useItem.tsx

```diff
@@ -282,7 +292,8 @@ export default function useItem(props: UseItemProps) {
     display: "flex",
     overflow: "hidden",
     alignItems: "stretch",
-    [paddingSide]: Math.max(0, -parseInt(itemStyle[side]?.toString() || "0")),
+    [paddingStart]: Math.max(0, -parseInt(itemStyle[sideStart]?.toString() || "0")),
+    [paddingEnd]: Math.max(0, -parseInt(itemStyle[sideEnd]?.toString() || "0")),
```

Comment:
try parsing to int like this:  
```ts
Math.max(0, -itemStyle[sideStart] || 0))
```

Reply:
`-` should convert a string to a number

Reply:
Also, not sure why I used `itemStyle[sideStart]` and not `deltaXStart`. same for end.  
Please try to use `deltaXStart` and `deltaXEnd` and see if it works.